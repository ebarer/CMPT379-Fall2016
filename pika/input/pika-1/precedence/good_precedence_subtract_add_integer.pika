exec {

  print 73 - 17 + 91, _n_.
  print 17 - 43 + 88, _n_.
  print 23 - 87 + 80, _n_.
  print 13 - 6 + 20, _n_.
  print 30 - 94 + 30, _n_.
  print 3 - 22 + 63, _n_.
  print 28 - 41 + 67, _n_.
  print 44 - 31 + 93, _n_.
  print 39 - 74 + 41, _n_.
  print 54 - 7 + 60, _n_.
  print 96 - 5 + 43, _n_.
  print 15 - 83 + 43, _n_.

  print 71 + 54 - 60, _n_.
  print 14 + 15 - 53, _n_.
  print 65 + 37 - 4, _n_.
  print 20 + 13 - 91, _n_.
  print 99 + 25 - 69, _n_.
  print 89 + 9 - 87, _n_.
  print 76 + 85 - 24, _n_.
  print 8 + 44 - 59, _n_.
  print 30 + 66 - 81, _n_.
  print 38 + 53 - 25, _n_.
  print 22 + 38 - 80, _n_.
  print 39 + 56 - 91, _n_.

  print 96 - 83 + 16 - 1, _n_.
  print 98 - 6 + 51 - 87, _n_.
  print 30 - 86 + 3 - 91, _n_.
  print 4 - 33 + 96 - 40, _n_.
  print 73 - 72 + 37 - 41, _n_.
  print 27 - 65 + 20 - 73, _n_.
  print 78 - 24 + 77 - 75, _n_.
  print 7 - 25 + 22 - 24, _n_.
  print 73 - 45 + 19 - 43, _n_.
  print 69 - 70 + 43 - 98, _n_.
  print 90 - 39 + 9 - 63, _n_.
  print 86 - 27 + 69 - 37, _n_.

  print 19 + 28 - 72 + 81, _n_.
  print 98 + 78 - 19 + 99, _n_.
  print 2 + 10 - 82 + 39, _n_.
  print 87 + 66 - 50 + 18, _n_.
  print 80 + 24 - 65 + 24, _n_.
  print 91 + 50 - 48 + 63, _n_.
  print 53 + 50 - 25 + 61, _n_.
  print 2 + 87 - 78 + 77, _n_.
  print 13 + 34 - 91 + 18, _n_.
  print 17 + 75 - 72 + 13, _n_.
  print 88 + 9 - 64 + 34, _n_.
  print 72 + 23 - 28 + 19, _n_.

}

