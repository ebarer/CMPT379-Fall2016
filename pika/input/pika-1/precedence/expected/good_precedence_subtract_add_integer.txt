147
62
16
27
-34
44
54
106
6
107
134
-25
65
-24
98
-58
55
11
137
-7
15
66
-20
4
28
56
-144
27
-3
-91
56
-20
4
-56
-3
91
56
256
-31
121
63
156
139
88
-26
33
67
86
